outputs.tf

```diff
@@ -1,3 +1,10 @@
+output "usage_IAM_roles" {
+  description = "Basic IAM role(s) that are generally necessary for using the resources in this module. See https://cloud.google.com/iam/docs/understanding-roles."
+  value = [
+    "roles/compute.instanceAdmin.v1",
```

Comment:
Wouldn't recommend `admin` role to be advertised by this module. 

If we need to assign this admin role to any infra piece, we can add it explicitly into the blueprint

Reply:
Okay, which role do you recommend?
It's for bastion host to access other VM in the project.